google/cloud/storage/bucket_metadata.h

```diff
@@ -630,6 +642,17 @@ class BucketMetadata : private internal::CommonMetadata<BucketMetadata> {
   //@{
   /**
    * @name Get and set the IAM configuration.
+   *
+   * @warning this is an Alpha feature of Google Cloud Storage, it is not
```

Comment:
Feature is no longer Alpha and is in Beta.

Reply:
Fixed, here and elsewhere.